docs/technical_reference/flowsheets/ion_exchange.rst

```diff
@@ -0,0 +1,148 @@
+    * Next, three variables are unfixed on the ion exchange model to allow for the model to solve for the new conditions (``dimensionless_time``, ``number_columns``, and the ``bed_depth``).
+    * Finally, the model is solved for these new conditions.
+    
+    Note that stopping at this point will likely result in a non-integer value for ``number_columns``. This value also corresponds to an optimal value for both ``bed_depth`` and 
+    ``dimensionless_time``. The user can optionally ensure the ``number_columns`` is an integer by rounding the ``number_columns`` to an integer value after 
+    the initial optimization solve, fixing ``number_columns`` to that value, and then re-solving the model to obtain the final values for ``bed_depth`` and ``dimensionless_time``.
+    This is the approach taken in the demonstration file.
+        
+Degrees of Freedom
+------------------
+
+The degrees of freedom for the flowsheet can change depending on the configuration options specified during the build. 
+Specifics for other configuration options are available in the :doc:`IX unit model documentation </technical_reference/unit_models/ion_exchange_0D>`.
+In this demonstration, the following variables are initially fixed for simulating the IX flowsheet (i.e., degrees of freedom = 0):
+
+    * Feed conditions (component flows, temperature, pressure)
+    * Langmuir equilibrium coefficient for target ion
+    * Resin capacity, diameter, porosity, and density
+    * Number of columns for the system
+    * Service flow rate (loading rate)
+    * The dimensionless time for the constant-pattern solution
+
+Flowsheet Specifications
+------------------------
+
+.. csv-table::
+   :header: "Description", "Value", "Units"
+
+   "Feed molar flowrate of water", "2777.5", ":math:`\text{mol}/\text{s}`"
+   "Feed molar flowrate of target ion", "0.125", ":math:`\text{mol}/\text{s}`"
+   "Feed temperature", "298.15", ":math:`\text{K}`"
+   "Feed pressure", "101325", ":math:`\text{Pa}`"
+   "Langmuir equilibrium coefficient", "0.7", ":math:`\text{dimensionless}`"
+   "Resin capacity", "3.0", ":math:`\text{mol/kg}`"
+   "Resin bulk density", "0.7", ":math:`\text{kg/L}`"
+   "Constant pattern dimensionless time", "1", ":math:`\text{dimensionless}`"
+   "Number columns", "4", ":math:`\text{dimensionless}`"
+   "Bed void fraction", "0.5", ":math:`\text{dimensionless}`"
+   "Bed depth", "1.7", ":math:`\text{m}`"
+   "Service flow rate (in bed volumes)", "15", ":math:`\text{1/hr}`"
+
+Future Refinements
+------------------
+
+The following modifications to the IX flowsheet are planned for development:
+
+    * Add examples of the Freundlich (Clark) ion exchange model.
+    * Improve auto-scaling of model for ease of use
+
+Code Documentation
+------------------
+
+* :mod:`watertap.examples.flowsheets.ion_exchange`
+
+References
+----------
+
+| LeVan, M. D., Carta, G., & Yon, C. M. (2019).
```

Comment:
It's odd to see these reference entries as intentionally multi-line, but that's my only nit-picky comment.